awx/ui_next/src/screens/WorkflowApproval/WorkflowApprovalList/useWsWorkflowApprovals.js

```diff
@@ -0,0 +1,61 @@
+import { useState, useEffect } from 'react';
```

Comment:
was does the Ws in `useWsWorkflowApprovals` stand for?

Reply:
That's the naming pattern we have for all of our websocket hooks.  They all start with `useWs` and then we concat the resource on the end.